7_react/react1/week2/hw-react-week2/src/components/Checkbox.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Fragment } from "react";
+import '../App.css';
+
+const Checkbox = props => {
+  const {
+    onChange,
+    data: { id, description, done }
+  } = props;
+  return (
+    <Fragment>
```

Comment:
`<Fragment>...</Fragment` is only needed if you need to return multiple elements.

There is also a shorthand version of it `<>...</>`
https://legacy.reactjs.org/docs/fragments.html

Reply:
Yes, I was experimenting with fragments and then discovered there's a new format 🤓Thanks.